android/src/main/java/com/tailscale/ipn/ui/localapi/LocalAPIClient.kt

```diff
@@ -33,26 +48,31 @@ class LocalApiClient {
     //         the corresponding request.  Cookies must be unique for each request.
     external fun doRequest(request: String, method: String, body: String, cookie: String)
 
+    protected val scope = CoroutineScope(Dispatchers.IO + Job())
```

Comment:
Recognizing that there has been, is, and will be a lot of flux over the next little while, is there a way to plumb [lifecycle-aware coroutine scope](https://developer.android.com/topic/libraries/architecture/coroutines#lifecyclescope) down into this client?

Reply:
+1 to lifecycle aware scopes!

Reply:
I've added a comment to remind me to do this.  A little experimentation with the activity scope yielded nothing but problems but I'm probably misunderstanding how coroutine scopes and contexts work.  I have to RTFM.  Technically, we're a single activity app, so this isn't critical - at least until we're aren't a single activity app...